search_events_app/tests.py

```diff
@@ -36,3 +40,38 @@ def test_event_model(self):
 
     def test_event_with_two_features(self):
         self.assertEqual(len(self.new_event.feature), 2)
+
+class TestApiService(TestCase):
+
+    @patch.object(
+        DummyApi,
+        'get',
+        return_value=[
+            {
+                'name': 'Carats world tour',
+                'organizer_name': 'Seung',
+                'country': 'Italy',
+                'url': 'https://www.eventbrite.com/e/carats-world-tour-tickets-102537931714?aff=ebdssbonlinesearch',
+                'start_date': '2020-07-20',
+                'features': ['EB Studio'],
+                'language': 'English',
+                'category': 'Music',
+                'format': 'Festival'
+            },
+            {
+                'name': 'Virtual stitch & bitch',
+                'organizer_name': 'Fashion Revolution',
+                'country': '',
+                'url': 'https://www.eventbrite.co.uk/e/fashion-question-time-tickets-90925824589?aff=ebdssbonlinesearch',
+                'start_date': '2021-01-30',
+                'features': ['Repeating event'],
+                'language': 'English',
+                'category': 'Fashion',
+                'format': 'Expo',
+            },
+        ]
```

Comment:
Maybe, this mock (and the next ones) should be in a different file and imported from there, just to keep a clear view of the tests.
What do you think about this?

Reply:
We thought the same thing but we didn't know how to do that. Thanks Romi